### ISSUE TITLE
Plugin enabled but do not see in Blueprint editor toolbar

### ISSUE THREAD

**Issue by basilwort5** [ZackBradshaw/Bluepy#1]
For ReadMe step under Usage: "Activate the plugin through the editor's toolbar."

In Plugins, I can see 'blupy', and it is ticked meaning enabled. I created a new Blueprint with parent of Object. In Blueprint editor, I cannot find where to activate the plugin. Could you add a screenshot? Or any settings I should double-check?

**Comment by AlfredMulder** [comment]
![image](https://github.com/ZackBradshaw/Bluepy/assets/36332878/8f6081fa-e2b3-4b61-97b3-b26549b3e582)
There is no “activate” button in blueprint editor's toolbar. Seems like error to me.

**Comment by basilwort5** [comment]
It kind of works, in map view (not blueprints) I can see a panel that opens with 'Click actor for more info', and 'Measure the distance between two points'. Don't know what that has to do with creating blueprints though.

**Comment by ZackBradshaw** [comment]
Hey folks thanks for your interest in this project.

Things are still under development and there were some difficulties with the feasibility of some features.
 
Things are still not fully flushed out on the unreal plugin side of things most of my efforts made so far have been on the model end of things though this poses a whole set of new challenges since the sear amount of token input and output for raw blueprint codeis quite large for the logic represented compared to more traditional languages begging the question as to is this still worth to pursue.

Possible ways to mitigate these issues
- Data is cleaned such that only unique fields are sent and hopefully received if fine-tuning goes correctly 🤞 
- Finetuning a long context model similar to this https://huggingface.co/gradientai/Llama-3-8B-Instruct-Gradient-4194k 
- Finetuning on a multi-modals (llava) so that only the image is required for context bringing down the compute and context reqs of the model

Unfortnately I lost access to the compute I was going to train this on so I'm looking at alternatives 

Hang tight and thanks again for your interest in this project 😃

**Comment by basilwort5** [comment]
Thanks for the update, I'll check back later. Hope you can make good progress, this sounds like it would be a very helpful plugin. I'll close this issue.

**Comment by basilwort5** [comment]
Thanks for the update, I'll check back later. Hope you can make good progress, this sounds like it would be a very helpful plugin. I'll close this issue.
